modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/FilterOrderControls.js

```diff
@@ -22,12 +22,11 @@ const FilterOrderControls = ({
 	sortingOrder,
 	sortingURL,
 }) => {
-	const showOrderToggle =
-		!orderDropdownItems || orderDropdownItems.length <= 1;
+	const showOrderToggle = orderDropdownItems && orderDropdownItems.length > 1;
 
 	return (
 		<>
-			{filterDropdownItems && (
+			{filterDropdownItems && !!filterDropdownItems.length && (
```

Comment:
the  !!filterDropdownItems.length was put by the SF it change it from  filterDropdownItems.length > 0
so....

if you want to change it, please feel free because for me is the same :D